ocean-client/src/vault-maxi.ts

```diff
@@ -206,12 +206,20 @@ export async function main(event: maxiEvent, context: any): Promise<Object> {
                 + ", " + (program.isSingle() ? ("minting only " + program.assetA) : "minting both"))
             let exposureChanged = false
 
+            if (!program.consistencyChecks(vault)) {
+                //maybe just temporary, try again
+                await delay(15000)                  
+                console.warn("consistency checks failed. maybe just a temporary caching topic. will try again")  
+                vault = await program.getVault() as LoanVaultActive
+                balances = await program.getTokenBalances()
+                pool = await program.getPool(program.lmPair)
```

Comment:
do we need balances and pool to do the consistencyChecks again?

Reply:
no, but since we had an inconsistency, if its solved after the wait, the balances are also to be refreshed -> refresh all.